### ISSUE TITLE
Linux get_interfaces_list not showing DOWN interfaces

### ISSUE THREAD

**Issue by k1tzu** [tomas/network#12]
It's because you use os.networkinterfaces()

**Comment by roccomuso** [comment]
+1

**Comment by tomas** [comment]
Yes, I'm aware of this. The only easy way around it would be to parse the output of `ifconfig`/`ipconfig`, which I'm not sure would be fully compatible because of the output format of each version.
